tests/out/wgsl/800-out-of-bounds-panic-vert.wgsl

```diff
@@ -7,11 +7,11 @@ struct VertexPushConstants {
 };
 
 struct VertexOutput {
-    [[location(0)]] frag_color: vec4<f32>;
-    [[builtin(position)]] member: vec4<f32>;
+    @location(0) @ frag_color: vec4<f32>;
+    @builtin(position) member: vec4<f32>;
 };
 
-[[group(0), binding(0)]]
+@group(0) @binding(0) 
```

Comment:
Extra space at end (replace artifact?)

Reply:
I think it's fine to have that extra space